.github/workflows/reusable-build.yml

```
@@ -0,0 +1,450 @@
+          profile: ${{ inputs.profile }}
+          plugin: false
+          pre: unset CC_x86_64_unknown_linux_gnu && unset CC # for jemallocator to compile
+
+      - name: Upload wasm artifact
+        id: upload-wasm-artifact
+        uses: ./.github/actions/artifact/upload
+        if: ${{ inputs.target == 'wasm32-wasip1-threads' && steps.check_cache.outputs.exists != 'true' && !inputs.skipable }}
+        with:
+          name: bindings-wasm32-wasi
+          path: crates/node_binding/rustbolt.wasm32-wasi.wasm
+          try-local-cache: ${{ inputs.profile == 'ci' }}
```

actionlint (1.7.4)

input "try-local-cache" is not defined in action "Upload Artifact" defined at "./.github/actions/artifact/upload". available inputs are "force-use-github", "name", "path"

(action)